lib/components/Select.js

```diff
@@ -15,25 +15,28 @@ class Select extends Component {
       value,
       onChange,
       leadingIcon = null,
+      infoText = "",
+      infoTextPosition = "right",
```

Comment:
```suggestion
      infoText = "",
```

Reply:
@goutham-subramanyam We are using these props in NeetoDesk. Should we remove it?